### ISSUE TITLE
'defaultIllustrationLibraryName' error on group creation

### ISSUE THREAD

**Issue by Steven-Harris** [sillsdev/rapuma#4]
I wanted to post this issue that I have had a couple of times. When creating a group of comp_type = usfm rapuma will generate this error

```
Traceback (most recent call last):
  File "/usr/local/bin/rapuma", line 901, in <module>
    userArguments(parser.parse_args())
  File "/usr/local/bin/rapuma", line 462, in userArguments
    ProjSetup(sysConfig, pid, gid).addGroup(cType, gid)
  File "/usr/local/lib/python2.7/dist-packages/rapuma/project/proj_setup.py", line 306, in addGroup
    aProject.createGroup()
  File "/usr/local/lib/python2.7/dist-packages/rapuma/manager/project.py", line 202, in createGroup
    groupObj = ManagerClass(self, cfg)
  File "/usr/local/lib/python2.7/dist-packages/rapuma/group/usfm.py", line 60, in __init__
    self.proj_illustration      = ProjIllustration(self.pid, self.gid)
  File "/usr/local/lib/python2.7/dist-packages/rapuma/project/proj_illustration.py", line 60, in __init__
    self.userIllustrationLibName = self.userConfig['Resources']['defaultIllustrationLibraryName']
  File "/usr/local/lib/python2.7/dist-packages/configobj.py", line 554, in __getitem__
    val = dict.__getitem__(self, key)
KeyError: 'defaultIllustrationLibraryName'

```

Note `defaultIllustrationLibraryName` is missing from my /config/rapuma.config. However, adding `defaultIllustrationLibraryName = Knowles-600_bw` to the bottom the [System] array does not fix the problem. 

The first time I had this problem I was running rapuma in a VM and now I am running rapuma out of /var/lib/rapuma directory. Sorry that is the only information I can give.

**Comment by thresherdj** [comment]
@megahirt - I think this parameter is no longer needed and I would like to remove it from the code. However, before doing that, I would like to see my personal branch merged into the master first but I'm not very good at doing that kind of thing, especially since there have been so many changes making a lot of differences between the two. Chris, would you be willing to do the merge? Or if you prefer some other method, that would be okay too. If you could do that, then I could go a head fix this issue on my personal branch and test it before moving it to the master.

**Comment by megahirt** [comment]
I'll try my best to do this on Thursday.  I'll let you know if I have
questions.

**Comment by thresherdj** [comment]
@megahirt - Thanks, I will not push anything until I see its been merged.

**Comment by thresherdj** [comment]
I have removed all references of defaultIllustrationLibrary... from the code as this was a dead-end and was only causing an error because one of the refs were misspelled. Once master has been updated/merged I'll push this to personal/dennis so it can be tested. Then this issue can be closed.

**Comment by megahirt** [comment]
I started the merge but didn't finish today.  Sorry.  Will resume tomorrow.

**Comment by Steven-Harris** [comment]
@thresherdj did the bug fix get pushed?

**Comment by thresherdj** [comment]
@Steven-Harris no, not yet, waiting on the merge. Once master is the same as personal/dennis, I'll push the changes to personal/dennis so they can be tested, though I don't anticipate any problems as the code I fixed was kind of a dead-end.

I'm dealing with two problems, first is the defaultIllustrationLibrary issue, which will be pushed soon. Then there is the issue of how Rapuma references its config file and what's in it. That will be a bit invasive and will need good testing.

**Comment by thresherdj** [comment]
Master is now updated and the defaultIllustrationLibrary issue has been dealt with. I need to test a bit then I can move on to the Rapuma config file issue.

**Comment by Steven-Harris** [comment]
Great! Thank you so much for taking care of this for me!
